src/elements/content-sidebar/versions/VersionsItemButton.js

```diff
@@ -53,9 +54,9 @@ class VersionsItemButton extends React.Component<Props> {
     };
 
     render() {
-        const { children, fileId, isCurrent, isDisabled, isSelected, onClick } = this.props;
+        const { children, fileId, isCurrent, isDisabled, isSelected, onClick, showVersionPreview } = this.props;
         const buttonClassName = classNames('bcs-VersionsItemButton', {
-            'bcs-is-disabled': isDisabled,
+            'bcs-is-disabled': showVersionPreview ? !showVersionPreview : isDisabled,
```

Comment:
We're drilling this prop multiple levels to add a visual treatment. Is this something we can accomplish by adding a CSS class much higher up in the DOM?

Reply:
great question. I will look into it! Would be great if we didn't need to  have this here. I was unhappy with it lol